linkage-monitor/src/main/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitorException.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.dependencies.linkagemonitor;
+
+/** Error when {@link LinkageMonitor} reports new linkage errors. */
```

Comment:
"Error when" --> "Exception thrown if" since this is not a java.lang.Error

Reply:
Updated

https://github.com/GoogleCloudPlatform/cloud-opensource-java/pull/710/files#diff-df70bea43d3a25e2c723b466bb150234R117